src/__tests__/helpers/rafPolyfill.js

```diff
@@ -0,0 +1,11 @@
+/*
+  React 16 expects reqeustAnimationFrame to be present,
+  including in test environments.
+
+  https://github.com/facebook/jest/issues/4545
+*/
+const raf = (global.requestAnimationFrame = callback => {
+  setTimeout(callback, 0);
+});
+
+export default raf;
```

Comment:
I rather use the [raf module](https://www.npmjs.com/package/raf), it provides a polyfill at "raf/polyfill" (see my note above about `jest.config.js`)